test/internal/src/main/java/com/android/designcompose/test/internal/Fonts.kt

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright 2023 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.android.designcompose.test.internal
```

Comment:
Why is this font file under a package name called .test.internal?

Reply:
The font needs to be shared between the test apps, so it needs to go into a common location. I have more shared test code that will be moved into this package in the future, especially related to Roborazzi and benchmarking.